tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/bytetype/BByteValueTest.java

```diff
@@ -564,7 +564,15 @@ public void testBitwiseUnsignedRightShiftOperator2() {
         invokeUnsignedRightShiftOperatorTestFunction(a, i, j);
     }
 
-    private void invokeUnsignedRightShiftOperatorTestFunction(byte a, long i, long j) {
+    @Test(description = "Test bitwise unsigned right shift operator 2")
+    public void testBitwiseUnsignedRightShiftOperator3() {
+        int a = 0xff;
+        long i = -23445834;
+        long j = 5;
+        invokeUnsignedRightShiftOperatorTestFunction(a, i, j);
```

Comment:
shall we test when it is a `const` as in the issue?

Reply:
Shall we also test what happens when it is a byte shifted by a large value? We need to see if it becomes zero.

Reply:
Shall we test the left shift cases also?

Reply:
Added test for byte shift with large value and becomes 0. Left shift cases are already added through PR. 

For constants, IMO since constants are compile-time, the constant operation should be work the same as the underlying type operation at runtime. Any issue with constant operation should be reproducible by a given type of operation. Adding a test would be redundant for me.

Reply:
The issue says `In both cases changing const to int makes the problem go away.` That's why I thought it might be an issue with the `const`. Don't you think it would be better to have atleast the test from the issue https://github.com/ballerina-platform/ballerina-lang/issues/31394